docs/guide/multisite.md

```diff
@@ -2,109 +2,189 @@
 
 ## Introduction
 
-With eZ Platform it is possible to serve multiple, completely different sites using one eZ Platform instance and database.
+You can serve multiple different sites using one eZ Platform instance and database.
 
-Each site will have its own content root, at a lower level than the default one (Location ID 2). One backoffice can be used for each site, but it is also possible to use a global one.
+Each site will have its own content root, at a lower level than the default one (Location ID 2).
+You can use one global back office for all sites, or a separate back office for each site.
 
-This feature is a reimplementation of the [PathPrefix](http://doc.ez.no/eZ-Publish/Technical-manual/4.x/Reference/Configuration-files/site.ini/SiteAccessSettings/PathPrefix) one that existed in eZ Publish Legacy.
+This feature is a reimplementation of the [PathPrefix](http://doc.ez.no/eZ-Publish/Technical-manual/4.x/Reference/Configuration-files/site.ini/SiteAccessSettings/PathPrefix) that existed in eZ Publish Legacy.
 
-## Configuration
+## Configuring multisite
 
-The feature is configured in `ezplatform.yml`, either at siteaccess or siteaccess group level:
+Multisite is configured in `ezplatform.yml`, either at [SiteAccess](siteaccess.md) or SiteAccess group level:
 
 ``` yaml
 ezpublish:
     system:
-        ezdemo_site_group:
+        site_group:
             content:
                 tree_root:
                     # Root locationId. Default is top locationId
                     location_id: 123
-                    # Every URL aliases starting with those prefixes will be considered
+                    # All URL aliases starting with those prefixes will be considered
                     # being outside of the subtree starting at root_location.
                     # Default value is an empty array.
                     # Prefixes are not case sensitive.
                     excluded_uri_prefixes: [ /media, /images ]
 ```
 
-##### `content.tree_root.location_id`
+### Configuration parameters
 
-This parameter sets the Location ID of the content root. The API is jailed within this root, and nothing above this level will be accessible by default. This parameter can be filtered using the `excluded_uri_prefixes` parameter described below.
+#### `location_id`
 
-##### `content.tree_root.excluded_uri_prefixes`
+`content.tree_root.location_id` sets the Location ID of the content root. The API is limited to ("jailed" within) this root, and nothing above this level will be accessible by default. This parameter can be filtered using the `excluded_uri_prefixes` parameter.
```

Comment:
This is incorrect actually.

"API" knows close to nothing about this, so there is no "jail". "Site handling" is done above in the architecture, so in controllers and view logic. In kernel often referred to as MVC layer, which actually does not deal with M:Model which is done by API, but it does deal with V:View and C:Controller. 

But that is to kernel specific here, we need a higher level way to describe it I think, it's been a topic of discussion before.

In this specific case you can say it like this perhaps: ``content.tree_root.location_id` sets the Location ID of the content root of your "site", it can be filtered using the `excluded_uri_prefixes` parameter. These settings are taken into account several places for you, for instance: Url Alias lookups for incomming web requests, and url alias path generation in twig templates. In addition you'll need to take these into account when generating site paths in your layout, or other places rendering site/tree structure.`


/cc @bdunogier